serializer/src/main/java/net/md_5/bungee/api/chat/hover/content/ItemSerializer.java

```diff
@@ -49,6 +49,7 @@ public Item deserialize(JsonElement element, Type type, JsonDeserializationConte
         return new Item(
                 ( value.has( "id" ) ) ? value.get( "id" ).getAsString() : null,
                 count,
+                ( value.has( "components" ) ) ? context.deserialize( value.get( "components" ), JsonElement.class ) : null,
```

Comment:
is deserialize required here as it already is a JsonElement, i tested it locally and all checks passed without it